deployment/tools/pluginIdMigration.py

```diff
@@ -31,14 +31,11 @@
             plugin["id"] = slugify(plugin["title"])
 else:
     print "No webportal plugins to convert, exit..."
-    sys.exit(0)
+    sys.exit(-1)
```

Comment:
I suggest to overwrite whether the file will change or not.

Reply:
Done